pixart.js

```diff
@@ -1 +1,80 @@
+var userInput = $("#color-field").val();;
 
+$("#set-color").on("click", function (evt){
+  evt.preventDefault();
+  var userInput = $("#color-field").val();
+  var brushBox = $(".brush").css({
+    background: userInput
+  });
+});
+$("#set-color").bind("keypress", function (evt){
```

Comment:
I prefer `keyup` but this works! http://stackoverflow.com/questions/3396754/onkeypress-vs-onkeyup-and-onkeydown